File: TransCoder_tokenized_test_set_functions/python/PRINT_POSSIBLE_STRINGS_CAN_MADE_PLACING_SPACES_2.py
def printSubsequences ( str ) :
    n = len ( str )
    opsize = int ( pow ( 2, n - 1 ) )
    for counter in range ( opsize ) :
        for j in range ( n ) :
            print ( str [ j ], end = "" )
            if ( counter & ( 1 << j ) ) :
                print ( " ", end = "" )
        print ( "\n", end = "" )
    

File: TransCoder_tokenized_test_set_functions/cpp/PRINT_POSSIBLE_STRINGS_CAN_MADE_PLACING_SPACES_2.cpp
void printSubsequences ( string str ) {
  int n = str . length ( );
  unsigned int opsize = pow ( 2, n - 1 );
  for ( int counter = 0;
  counter < opsize;
  counter ++ ) {
    for ( int j = 0;
    j < n;
    j ++ ) {
      cout << str [ j ];
      if ( counter & ( 1 << j ) ) cout << " ";
    }
    cout << endl;
  }
}